File: C/2020/12限/func2.c
//  func2.c
//  cl  func2.c
#include <stdio.h>

void add(int ii, int jj);
void add(int ii, int jj) {
    ii = ii + jj;
    printf("add: %d \n", ii);
}

int main() {
    int i = 10;
    int j = 20;
    add(i, j);
    printf("main: %d \n", i);

    return 0;
}

File: GameServer/2022/htdocs(34)/func2.php
<?php   // func2.php

function 関数名($引数) {
    echo "in function　{$引数} <br>";
}
//
関数名(1);
関数名(999);
関数名("str");
echo "<br> end <br>";

function 関数名2($引数1, $引数2) {
    echo "in func2　{$引数1}, {$引数2} <br>";
}
関数名2(1, 999);
関数名2("1st", "2nd");
